obs-studio-client/source/controller.hpp

```diff
@@ -33,17 +34,38 @@ struct ProcessInfo
 {
 	uint64_t handle;
 	uint64_t id;
+	DWORD    exit_code;
 
-	ProcessInfo()
+	enum ExitCode
```

Comment:
I think we can add this to lib-streamlabs-ipc , as it's more generic, but did not know where to put it exactly, so I left it here for now.

Reply:
Agree, seems better to have them in lib-streamlabs-ipc especially given the fact we are returning these specifics error codes from the server below. I would put them in ipc.h